### PR TITLE
[FRT-781] Deprecate Artifact class field and expose Description and link fields

### PR DESCRIPTION
**Pull request by pgaref**
Release Notes
-------------

Bug Fixes
- Deprecate Artifact class field and expose Description and link fields

Checklist
---------
- [X] Leave this box unchecked if features are not yet available in production

### CODE REVIEW

File: internal/flink/command_artifact.go [hunk 1]
```diff
@@ -13,11 +13,12 @@ type flinkArtifactOut struct {
 	Id            string `human:"ID" serialized:"id"`
 	Name          string `human:"Name" serialized:"name"`
 	Version       string `human:"Version" serialized:"version"`
-	Class         string `human:"Class" serialized:"class"`
 	Cloud         string `human:"Cloud" serialized:"cloud"`
 	Region        string `human:"Region" serialized:"region"`
 	Environment   string `human:"Environment" serialized:"environment"`
 	ContentFormat string `human:"Content Format" serialized:"content_format"`
+	Description   string `human:"Description" serialized:"description"`
+	DocLink       string `human:"Documentation link" serialized:"doc_link"`
```
Comment (sgagniere) [inline internal/flink/command_artifact.go:21]: We should make it consistent between the field name, the human tag, and the serialized tag.

And we prefer not to abbreviate single words, so maybe `DocumentationLink`, `Documentation Link` and `documentation_link`

File: internal/flink/command_artifact_create.go [hunk 2]
```diff
@@ -56,8 +59,8 @@ func (c *command) newCreateCommand() *cobra.Command {
 	pcmd.AddRegionFlagFlink(cmd, c.AuthenticatedCLICommand)
 	pcmd.AddEnvironmentFlag(cmd, c.AuthenticatedCLICommand)
 	cmd.Flags().String("runtime-language", "java", fmt.Sprintf("Specify the Flink artifact runtime language as %s.", utils.ArrayToCommaDelimitedString(allowedRuntimeLanguages, "or")))
-	cmd.Flags().String("description", "", "Description of Flink artifact.")
-	cmd.Flags().String("class", "default", "Name of Flink artifact class or alias.")
+	cmd.Flags().String("description", "", "Specify the Flink artifact Description.")
+	cmd.Flags().String("documentation-link", "", "Specify the Flink artifact Documentation link.")
```
Comment (sgagniere) [inline internal/flink/command_artifact_create.go:63]:
```suggestion
	cmd.Flags().String("description", "", "Specify the Flink artifact description.")
	cmd.Flags().String("documentation-link", "", "Specify the Flink artifact documentation link.")
```

File: internal/flink/command_artifact.go [hunk 3]
```diff
@@ -13,11 +13,12 @@ type flinkArtifactOut struct {
 	Id            string `human:"ID" serialized:"id"`
 	Name          string `human:"Name" serialized:"name"`
 	Version       string `human:"Version" serialized:"version"`
-	Class         string `human:"Class" serialized:"class"`
```
Comment (sgagniere) [inline internal/flink/command_artifact.go:16]: This command is currently not GA, right?

If it is GA, then we can't remove fields yet.

Comment (pgaref) [reply internal/flink/command_artifact.go:16]: Thats correct, its not GA yet.

**Comment by pgaref** [comment]
Thanks for the comments @sgagniere ! Update the PR, please let me know what you think :)

**Review by sgagniere — APPROVED** [review]

**Pull request merged**